src/client.ts

```diff
@@ -24,8 +24,8 @@ export class Client {
     Log.info(chalk.yellow(`➡️ Signing in to ${host.baseUrl}...`));
```

Comment:
Could we update the log messages to use the `fullUrl` as well?

Reply:
I thought about this, it might represent a small change to what people expect that I tried to avoid - but it's not difficult to change.

Reply:
Thanks! It looks a little noisier but I think it'll clear confusion if there are ever two hosts on the same `baseUrl`